src/App.tsx

```diff
@@ -1,10 +1,67 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
+  FC, useEffect, useMemo, useState,
+} from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { ErrorType } from './types/Error';
+import { TodoList } from './components/TodoList/TodoList';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer';
+import { Filter } from './types/FilterConditions';
+
+const USER_ID = 10304;
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<ErrorType>(ErrorType.None);
```

Comment:
it's not a `boolean` type
```suggestion
  const [error, setError] = useState<ErrorType>(ErrorType.None);
```